lib/ui/src/core/shortcuts.js

```diff
@@ -86,11 +86,7 @@ export default function initShortcuts({ store }) {
           if (!showNav) {
             fullApi.toggleNav();
           }
-          const element = document.getElementById('storybook-explorer-menu');
-
-          if (element) {
-            element.focus();
-          }
+          fullApi.focusOnUIElement('storyListMenu');
```

Comment:
can you pull these constants out to be shared between modules?

Reply:
Sure thing, on it. 👍

Reply:
➕

Reply:
Hmmm then do you want me to just export them for now?